netutils/config/clean.py

```diff
@@ -52,12 +52,12 @@ def clean_config(config, filters):
     return config
 
 
-def sanitize_config(config, filters):
+def sanitize_config(config: str, filters: t.Optional[t.List[t.Dict[str, str]]] = None) -> str:
```

Comment:
Why set it to none?

Reply:
As per docstring this needs to default to empty and due to the mutability of default arguments we cannot default to an empty list, therefore None and this:

```
if not filters:
    filters = []
```

Reply:
I would change the doc string to meet the code, not the other way around

Reply:
Sorry I missed this one, because of this: https://docs.python-guide.org/writing/gotchas/